extensions/interactions/rulesSpec.js

```diff
@@ -13,13 +13,13 @@
 // limitations under the License.
 
 /**
- * @fileoverview Unit tests to check that all the relevant rule specs exist.
+ * @fileoverview Unit tests to check that all the relevant rules exist.
  */
 
 describe('Rule spec services', function() {
-  var ruleSpecsServices = {};
-  var rulesJson;
-  var CLASSIFIER_RULESPEC_STR = 'FuzzyMatches';
+  var rulesServices = {};
+  var ruleTemplates;
+  var CLASSIFIER_RULE_STR = 'FuzzyMatches';
```

Comment:
I wonder if we could $inject this (using $injector, similar to below)? No worries if not.